### PR TITLE
Updated bulk demo with failure condition

### PR DESCRIPTION
**Pull request by chandrams**
Updated bulk demo with the following:

- Exit the loop when job FAILED
- Log messages
- Moved into a folder - bulk_demo
- TFB not deployed by default

### CODE REVIEW

**Comment by chandrams** [comment]
@dinogun - I 'm capturing the recommendations in recommendations_data.json, should I turn off printing the recommendations on stdout ?

File: monitoring/local_monitoring/bulk_demo/Bulk_demo.md [hunk 1]
```diff
@@ -10,14 +10,13 @@ recommendations. It also has settings to generate recommendations at both the co
 Bulk returns a `job_id` as a response to track the job status. The user can use the `job_id` to monitor the
```
Comment (dinogun) [inline monitoring/local_monitoring/bulk_demo/Bulk_demo.md:10]: Its a good idea to move this to a separate dir ... however please rename this as README.md and also add a comment in the local_monitoring README about this demo as well

Comment (chandrams) [reply monitoring/local_monitoring/bulk_demo/Bulk_demo.md]: Done

File: monitoring/local_monitoring/bulk_demo/README.md [hunk 2]
```diff
@@ -39,11 +38,12 @@ cd kruize-demos/monitoring/local_monitoring
 ```
```
[the remark this answers is not in the document]
Comment (chandrams) [reply monitoring/local_monitoring/bulk_demo/README.md:38]: Done

**Review by dinogun — APPROVED** [review]
LGTM

**Pull request merged**